src/CONST.js

```diff
@@ -2596,6 +2596,10 @@ const CONST = {
             NAVIGATE: 'NAVIGATE',
         },
     },
+    TIME_PERIOD: {
```

Comment:
Might be out of scope for this PR, but I wonder if this should be localised as some other languages don't really use AM/PM.

Reply:
I'm not fluent in Spanish 🥲 . As far as I know, these are Latin shortcuts, and they should also work for Spanish. But it would be better to ask someone.

Reply:
I don't display CONS for users; it's strictly for developer use. 
Regarding the am/pm issue, @stitesExpensify, could you please ask carlos about it?

Reply:
I believe am/pm is widely used in Spanish. We can leave it for now